src/calendar-heatmap.js

```diff
@@ -167,23 +244,35 @@ function calendarHeatmap() {
           })
           .attr('x', function (d, i) {
             var matchIndex = 0;
+            var monthSpacing =  0;
+            if (monthSpace){
+              monthSpacing = moment(d).diff((firstDate).startOf('month'), 'months');
+            }
             dateRange.find(function (element, index) {
               matchIndex = index;
               return moment(d).isSame(element, 'month') && moment(d).isSame(element, 'year');
             });
-
-            return Math.floor(matchIndex / 7) * (SQUARE_LENGTH + SQUARE_PADDING);
+            return ((Math.floor(matchIndex / 7)+monthSpacing) * (SQUARE_LENGTH + SQUARE_PADDING));
           })
           .attr('y', 0);  // fix these to the top
 
       days.forEach(function (day, index) {
-        if (index % 2) {
+        if(allWeekdayNames){
```

Comment:
I think it should be all or nothing. So if `allWeekdayNames` if false, we should not show any weekday names